examples/for-learning-and-testing/terraform-test/{{ .ModuleName | snakecase }}_test.go

```diff
@@ -0,0 +1,23 @@
+package test
```

Comment:
Having a file with `{{` in git will break the git clone in Windows... 

So far I thought about generation of files at runtime in "unix" tests:
* have generic file name like "module_test.go.template"
* in "unix" test copy the test directory to temp dir
* rename file to "{{..." 
* run tests

I can handle this in my windows CICD branch

Reply:
Yes, I saw that. Windows is always such a PITA :-D 

The one catch with code-generating this example is that I'm linking to the example from the new blog post... So if it's code-generated, that doesn't quite work... Is there any other alternative? Does it work with Cygwin or something?